src/main/kotlin/com/github/denofevil/aurelia/Aurelia.kt

```diff
@@ -12,16 +12,14 @@ object Aurelia {
     var REPEAT_FOR = "repeat.for"
     var AURELIA_APP = "aurelia-app"
 
-    fun present(project: Project): Boolean {
```

Comment:
nice, these were my first lines of (Java) code compiled with Kotlin compiler ;). Actually I suspected, that my PR might be rejected because of writing Java instead of Kotlin ;)